iotfunctions/aggregate.py

```diff
@@ -1,8 +1,13 @@
-# Licensed Materials - Property of IBM
-# 5737-M66, 5900-AAA, 5900-A0N, 5725-S86, 5737-I75
-# (C) Copyright IBM Corp. 2020 All Rights Reserved.
-# US Government Users Restricted Rights - Use, duplication, or disclosure
-# restricted by GSA ADP Schedule Contract with IBM Corp.
+# *****************************************************************************
```

Comment:
This text is completely different from the one earlier you added.
Do I need to do it my copyright changes too ?

Reply:
IOTfunctions is publicly available for download because it acts as our development client for customers. Therefore this open-source header. Analytics-service-pipeline , for example, is internal code which is not available to the public.